twisted/plugins/vumi_http_retry.py

```diff
@@ -0,0 +1,8 @@
+from twisted.application.service import ServiceMaker
+
+
+serviceMaker = ServiceMaker(
+    'vumi_http_retry',
+    'vumi_http_retry.service',
+    'vumi_http_retry',
```

Comment:
Maybe the description should be a little more verbose? (if only to reduce the number of occurrences of `vumi_http_retry`) :)